src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -1244,14 +1244,16 @@ protected SCMRevision retrieve(@NonNull String headName, @NonNull TaskListener l
                                 pr, headName, strategy == ChangeRequestCheckoutStrategy.MERGE
                         );
                         PullRequestSCMRevision prRev = createPullRequestSCMRevision(pr, head, listener, github, ghRepository);
-                        prRev.validateMergeHash(ghRepository);
 
                         switch (strategy) {
                             case MERGE:
-                                if (Boolean.FALSE.equals(pr.getMergeable())) {
```

Comment:
This check moved to part of validate as an exception.  Then this particular client catches and reports the error.